client/extensions/woocommerce/app/settings/shipping/shipping-class-list-entry.js

```diff
@@ -0,0 +1,76 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+
+import { editShippingClass } from 'woocommerce/state/ui/shipping/classes/actions';
+import Button from 'components/button';
+
+const ShippingClassEntry = ( { name, slug, description, loaded, translate, edit, isNew } ) => {
+	const cssClasses = [ 'shipping__classes-row' ];
+
+	if ( ! loaded ) {
+		cssClasses.push( 'is-placeholder' );
```

Comment:
`classnames` library would help a bit, to limit the verbosity.

Reply:
I switched to it.